src/azure-cli/azure/cli/command_modules/servicefabric/tests/latest/test_sf_managed_application.py

```diff
@@ -22,9 +22,9 @@ def test_managed_app_type(self):
             'vm_password': self.create_random_name('Pass@', 9),
             'app_type_name': 'VotingType',
             'v1': '1.0.0',
-            'app_package_v1': 'https://azclistaccounttest.blob.core.windows.net/azclicont/Voting.sfpkg?sp=r&st=2024-03-11T19:56:24Z&se=2024-04-07T03:56:24Z&sv=2022-11-02&sr=b&sig=hi56byw0jYpSLnJtMNemqHe92cZTMJgMHyeGa7GtN9E%3D',
+            'app_package_v1': 'https://sfmccliapptest.blob.core.windows.net/votingcont/Voting.sfpkg?sp=racwdyti&st=2024-10-28T05:39:21Z&se=2024-11-09T14:39:21Z&sv=2022-11-02&sr=b&sig=%2Fla1dP9YB3rLuyi5Nb0i6XDtTZSWb16IiUA%2BA7PLy4Q%3D',
```

Comment:
because the storage account with the package was deleted

Reply:
Why not consider dynamically creating storage accounts in the test? Otherwise, there is still a risk that this storage account will be deleted in the future